### PR TITLE
python3Packages.fontbakery: fix dependecies

### PR DESCRIPTION
**Pull request by jopejoe1**
## Description of changes

updated and fixed fontbakery dependencies so that fontbakery can build again.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by jopejoe1** [comment]
@ofborg eval python3Packages.beziers python3Packages.vfblib python3Packages.bablefont python3Packages.fontbakery

File: pkgs/development/python-modules/vfblib/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  build-system = [ setuptools ];
+
+  dependencies = [
+    fonttools
+    typing-extensions
+    ufonormalizer
+    ufolib2
+    defcon
+  ];
+
```
Comment (natsukium) [inline pkgs/development/python-modules/vfblib/default.nix:35]: Could you run tests with `pytestCheckHook` and `pythonImportsCheck`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Comment (jopejoe1) [reply pkgs/development/python-modules/vfblib/default.nix:36]: Sure

**Review by danc86 — APPROVED** [review]
Thanks for fixing these up!

**Comment by natsukium** [comment]
@ofborg build python312Packages.fontbakery

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 330084` at 1c6d14cc run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages built successfully:</summary>
  <ul>
    <li>fontbakery (python312Packages.fontbakery)</li>
    <li>python311Packages.babelfont</li>
    <li>python311Packages.beziers</li>
    <li>python311Packages.collidoscope</li>
    <li>python311Packages.fontbakery</li>
    <li>python311Packages.fontfeatures</li>
    <li>python311Packages.glyphtools</li>
    <li>python311Packages.vfblib</li>
    <li>python312Packages.babelfont</li>
    <li>python312Packages.beziers</li>
    <li>python312Packages.collidoscope</li>
    <li>python312Packages.fontfeatures</li>
    <li>python312Packages.glyphtools</li>
    <li>python312Packages.vfblib</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]
LGTM, thanks.

**Pull request merged**